src/Stryker.DataCollector/Stryker.DataCollector/Stryker.DataCollector.csproj

```diff
@@ -2,9 +2,9 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <AssemblyVersion>0.16.1.0</AssemblyVersion>
-    <FileVersion>0.16.1.0</FileVersion>
-    <Version>0.16.1</Version>
+    <AssemblyVersion>0.22.9.0</AssemblyVersion>
```

Comment:
In #1681 @richardwerkman proposes to simply remove all these version numbers since they are not used anywhere anyway. They would default and remain static but this should not really be a problem afaik.

Reply:
done
I rebased the branch to master head as it was behind